salt/_pillar/metalk8s.py

```diff
@@ -3,6 +3,8 @@
 import salt.utils.files
 import salt.utils.yaml
 
+from . import utils
```

Comment:
Maybe worth moving into the builtin `__utils__`

Reply:
Yup, done.